pages/_news/2025-02-09-BHEUPreprint.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "BioHackEU24 report: Bioschemas for Mortals"
+tags:
+- BHEU2024
+- Preprint
+- BioHackrXiv
+---
+
+We are pleased to announce the first [BioHackrXiv](https://biohackrxiv.org) preprint of 2025. It reports on the work done during the past [BioHackathon Europe 2024](https://biohackathon-europe.org/) by the group [Bioschemas for Mortals](https://github.com/elixir-europe/biohackathon-projects-2024/blob/main/10.md).
+
+The goal of this project is to reimagine, reframe and supplement the existing Bioschemas guidance available. A core team are progressing the tasks highlighted in the report. This will happen at a dedicated hackathon in Cologne, Germany between 17-19 February 2025.
+
+
+Citation and link:
+
+Reed P, Schnitzel H, and Juty N. BioHackEU24 report: Bioschemas for Mortals. OSF; 2025. doi:10.37044/osf.io/dch6w. Available: [https://osf.io/dch6w_v2/](https://osf.io/dch6w_v2/)
```

Comment:
I would suggest stating BioHackrXiv instead of OSF in the reference.

Reply:
[Like this?](https://github.com/BioSchemas/bioschemas.github.io/pull/831/commits/a1a4785e2d0bce6d662f38200b00219f0ee5eb25) 

I replaced 'OSF' with 'BioHackrXiv' just before the year.